vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -129,13 +137,14 @@
      * or an invalid Validation containing an accumulated List of errors.
      * @throws NullPointerException if values is null
      */
-    static <E, T> Validation<Seq<E>, Seq<T>> sequence(Iterable<? extends Validation<Seq<E>, T>> values) {
+    static <E, T> Validation<E, Seq<T>> sequence(Iterable<? extends Validation<E, T>> values) {
```

Comment:
I don't know why we did these generics. It should be

```java
sequence(Iterable<? extends Validation<? extends E, ? extends T>> values)
```

(see Either.sequence for example)